30413/Arpad Alex Lacatus/Lab07/extra5

```diff
@@ -0,0 +1,96 @@
+#include <stdio.h>
```

Comment:
What's up with this?

Reply:
I sent you the code for the extra problem. Is there a problem?